keyboards/om60/encoder.c

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2021 3araht
```

Comment:
Please remove this file, and use encoder map instead. 

https://docs.qmk.fm/#/feature_encoders?id=encoder-map

And via does support this.